adapter/integration-test/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "mock": "node src/mock.server.js",
     "pretest": "npm run mock &",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
Do we need this?

Reply:
I changed that for local debugging and will change back. I also think we need faster ci runs

Reply:
Allright :) yeah I'm also a bit concerned about the ci times, feels like way too much - but most of it results from docker artifact up- and download..